bcda/web/api.go

```diff
@@ -110,16 +111,30 @@ func bulkRequest(t string, w http.ResponseWriter, r *http.Request) {
 		err error
 	)
 
-	db := database.GetGORMDbConnection()
-	defer database.Close(db)
-
 	if ad, err = readAuthData(r); err != nil {
 		oo := responseutils.CreateOpOutcome(responseutils.Error, responseutils.Exception, "", responseutils.TokenErr)
 		responseutils.WriteError(oo, w, http.StatusUnauthorized)
 		return
 	}
 
+	db := database.GetGORMDbConnection()
+	defer database.Close(db)
+
 	acoID := ad.ACOID
+
+	var jobs []models.Job
+
+	// if we really do find this record
```

Comment:
Any chance we can beef up the commentary here to describe what we're doing here, and why we're doing it?